### DISCUSSION TITLE
Show borders of all protected area

### DISCUSSION THREAD

**Discussion by moonraver** [osmandapp/OsmAnd#17856] [Maps Related]
Hello,
I see that Osmand offline vector maps can only show borders of "Nature reserve". I is fine but it is only a small part from all types of protected areas. When I use Osmand for mapping I need to change map to "Osmand online tiles map=OSM" to see if I am still in protected area. It would be very helpful for mappers, hikers and bikers to see at least borders of a bigger protected areas in your offline vector maps like protected landscape areas or national parks.
Thank you

**Comment by DmitryAlexei** [comment]
This issue https://github.com/osmandapp/OsmAnd/issues/16976 has already been done.
You may need to update your app to the latest version available in Google Play.

**Reply by moonraver** [reply]
Version from Google play is old. I use only nightly builds released 3x daily with v2 OpenGL. Latest now is 4.6.0#590m. It shows only nature reserve in v1 and v2. For example: I need to see borders of mountain area called Biele Karpaty in Slovakia. It is a protected landscape area. After finding it, it shows only a point somewhere in these mountains and not the whole border. The same as mountain area Male Karpaty, and so on. I think this issue has NOT been already done. There were suggestions to make some GPX from overpass turbo and import it to map to see borders bud this is not a solution too. I think, to see protected areas is a MUST HAVE feature for hikers and bikers.

**Reply by moonraver** [reply]
I found out that I was looking for "area" via search button and it can only find POI (=point, not an area). I realized that when I tap on the name of the protected area or mountain, I can see it's borders. But it is still a pity that borders of all protected areas can not be seen directly in the map just like nature reserve. Will you ever plan this feature?
